cfgov/cfgov/settings/base.py

```diff
@@ -447,10 +446,11 @@
 ENABLE_AKAMAI_CACHE_PURGE = os.environ.get("ENABLE_AKAMAI_CACHE_PURGE", False)
 if ENABLE_AKAMAI_CACHE_PURGE:
     WAGTAILFRONTENDCACHE["akamai"] = {
-        "BACKEND": "v1.models.caching.AkamaiBackend",
+        "BACKEND": "cdntools.backends.AkamaiBackend",
         "CLIENT_TOKEN": os.environ.get("AKAMAI_CLIENT_TOKEN"),
         "CLIENT_SECRET": os.environ.get("AKAMAI_CLIENT_SECRET"),
         "ACCESS_TOKEN": os.environ.get("AKAMAI_ACCESS_TOKEN"),
+        "HOSTNAMES": ["www.consumerfinance.gov"],
```

Comment:
Is this a problem for beta.consumerfinance.gov, which has its own Akamai configuration?

Reply:
Moved this to a variable [in base.py](https://github.com/cfpb/consumerfinance.gov/pull/8522/files#diff-45fabeb89b7e2fdf7d0370b5207df871633e4994776f0883696844eadabbafdeR453) for both Akamai and Cloudfront. Will have a corresponding Ansible PR presently.